docs/rst/user_manual/chart_panel_index.rst

```diff
@@ -7,13 +7,42 @@
 Main Panel
 ##########
 
-In the central panel or *chart panel*, is where the data of the entities will be displayed.
-The main feature of the *Fast DDS Monitor* application is to graphically display the data that is being monitored.
+In the central panel, there is a tan section that allows multiple views, including a collapsed menu that reports the
+possible problems that have been detected on the DDS entities.
+
+The main feature of the *Fast DDS Monitor* application is to graphically display the data that is being monitored in
```

Comment:
Not sure if now we should say that it is the main feature.

Reply:
It was until now, in fact this line was copied from [previous version section](https://github.com/eProsima/Fast-DDS-monitor/blob/02c46bee11c2cfd11eef58988a418d4c449a188e/docs/rst/user_manual/chart_panel_index.rst?plain=1#L11). I don't think the graph views are the new main feature